File: Practice/21/C++/21.cpp
#include <string>
#include <iostream>
#include <cmath>
using namespace std;
int main()
{
    setlocale(LC_ALL, "Russian");
    double weight, height, BMI;
    cin >> weight >> height;
    BMI = weight / pow(abs(height) / 100, 2);
    if (BMI < 18.5) {
        cout << "Underweight";
    }
    else if (18.5 <= BMI < 25.0) {
        cout << "Normal weight";
    }
    else if (25.0 <= BMI < 30.0) {
        cout << "Overweight";
    }
    else if (30.0 <= BMI) {
        cout << "Obesity";
    }
    return 0;

}

File: Practice/21/Python/21.py
weight, height = map(float, input("Вес и рост:").split())
bmi = (abs(weight) / ((abs(height) / 100) ** 2))
if bmi < 18.5:
    print("Underweight")
elif 18.5 <= bmi < 25.0:
    print("Normal")
elif 25.0 <= bmi < 30.0:
    print("Overweight")
elif 30.0 <= bmi:
    print("Obesity")

